org.adempiere.base/src/org/compiere/model/MAuthorizationCredential.java

```diff
@@ -106,10 +106,33 @@ public String processToken(String code, MPInstance pinstance) {
 				msg = "Process instance parameter for Scopes not found";
 				return msg;
 			}
+			MPInstanceLog pilog = pinstance.addLog(null, 0, null, null, MAuthorizationAccount.Table_ID, 0);
+			msg = processToken(Env.getCtx(), code, paramScope.getP_String(), pilog);
+			pilog.saveEx();

```

Comment:
This line should be moved inside processToken "if  (pilog != null)", here it is not clear why you're saving if nothing is changed. Looks also as a side effect of processToken, better to keep it there